modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -52,6 +46,22 @@ func NewKeeper(cdc codec.BinaryCodec, key storetypes.StoreKey, authority string)
 	}
 }
 
+// NewKeeperWithConfig creates a new Keeper instance with the provided Wasm configuration.
+// This constructor function is meant to be used when the chain does not use x/wasm
+// and a Wasm VM needs to be instantiated using the provided parameters.
+func NewKeeperWithConfig(
+	cdc codec.BinaryCodec,
+	key storetypes.StoreKey,
+	authority string,
+	wasmConfig types.WasmConfig,
+) Keeper {
+	vm, err := cosmwasm.NewVM(wasmConfig.DataDir, wasmConfig.SupportedFeatures, types.ContractMemoryLimit, wasmConfig.ContractDebugMode, wasmConfig.MemoryCacheSize)
+	if err != nil {
+		panic(err)
+	}
+	return NewKeeperWithVM(cdc, key, authority, vm)
```

Comment:
nit: breathing room 🧘‍♂️ 

```suggestion
	}
	
	return NewKeeperWithVM(cdc, key, authority, vm)
```